src/object.c

```diff
@@ -754,10 +756,12 @@ size_t objectComputeSize(robj *o, size_t sample_size) {
             d = ((zset*)o->ptr)->dict;
             zskiplist *zsl = ((zset*)o->ptr)->zsl;
             zskiplistNode *znode = zsl->header->level[0].forward;
-            asize = sizeof(*o)+sizeof(zset)+(sizeof(struct dictEntry*)*dictSlots(d));
+            int dicthtNum = dictIsRehashing(d) + 1;
+            asize = sizeof(*o)+sizeof(zset)+dicthtNum*sizeof(dictht)
```

Comment:
Same as above. But there is an issue here. It does not include sizeof(dict) and sizeof(zskiplist). You may want to add those here.
Please refer to
https://github.com/antirez/redis/blob/unstable/src/server.h#L775-L778
https://github.com/antirez/redis/blob/unstable/src/object.c#L217-L226